src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -266,6 +276,58 @@ <h2 *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus ===
                     </div>
                 </ng-template>
             </div>
+
+            <!-- More Feedback Requests -->
+            <h2 *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus === COMPLETED">
+                {{ 'artemisApp.tutorExerciseDashboard.moreFeedback' | translate }}
+            </h2>
+            <div class="table-responsive" *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus === COMPLETED">
+                <table class="table table-striped exercise-table" *ngIf="moreFeedbackRequests?.length > 0; else noComplaints">
+                    <thead>
+                        <tr>
+                            <th>#</th>
+                            <th>{{ 'artemisApp.tutorExerciseDashboard.submissionDate' | translate }}</th>
+                            <th>{{ 'artemisApp.tutorExerciseDashboard.result' | translate }}</th>
+                            <th>{{ 'artemisApp.tutorExerciseDashboard.status' | translate }}</th>
+                            <th>{{ 'artemisApp.tutorExerciseDashboard.action' | translate }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let moreFeedbackRequest of moreFeedbackRequests; let i = index">
+                            <td>{{ i + 1 }}</td>
+                            <td>{{ moreFeedbackRequest.submittedTime | date: 'MMM d, y HH:mm:ss' }}</td>
+                            <td>
+                                <jhi-result [result]="moreFeedbackRequest.result"></jhi-result>
+                            </td>
+                            <td>
+                                {{ calculateMoreFeedbackStatus(moreFeedbackRequest.accepted) }}
```

Comment:
why are you using a function here? Could we not use `*ngIf`?

Reply:
same for calculateComplaintStatus above. Please avoid function calls in HTML because they are constantly evaluated